scripts/test_pg_dump.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Actually what does this test do that the pg_dump regression tests doesn't do and why can't we do this in a regression test?

Reply:
the regression test are all sql while this runs commands as different users and runs pg_dump at the end. but i will investigate if it could be part of regression test